CHANGELOG.md

```diff
@@ -26,6 +26,7 @@
 * Fixed layout bug where popovers triggered from a parent `Panel` with `modalSupport` active could
   render beneath that parent's own modal dialog.
 * Fixed broken `CodeInput` copy-to-clipboard feature.
+* Fixed `fetchJson` and related methods throwing on non-204 successful results without contents.
```

Comment:
56.5.0 has sailed - need to move up to the snapshot block / catch up with `develop`